com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/luaparser/LuaLexer.g4

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
The ANTLR grammar file for Lua (together with LuaParser.g4). These are used to generate the *.java files in `com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/luaparser/*.java` that we use to parse Lua files.